client/src/common/styledComponents/List.js

```diff
@@ -4,10 +4,14 @@ import styled from 'vue-styled-components';
 export const List = styled.div`
   display: flex;
   flex-wrap: wrap;
+
+  & > * {
```

Comment:
Is it possible to write it in some more readable way? Or create a separate styled component for that?

Reply:
Sorry it'is from @mrproper1337 request that already merged, I just check out from there